docs/guides/general-terminology/How-to-write-professional-commits.mdx

```diff
@@ -4,23 +4,70 @@ title: How to write professional commits
 
 # How to write professional commits ?
 
-To write professional commits in GitHub, it's important to follow a few best practices:
+To write professional commits in GitHub, it's important to follow a few best practices.
+
+## The conventional way:
+
+` type` `scope ` : `description`
+
+- `type` : Commit Type like build, ci, docs, feat, fix etc. This field is mandatory.
+
+- `scope` : Phrase describing a section of the codebase. This field is optional.
+
+- `description` : description should be in the present tense. Not capitalized. No period in the end. Each description consists of a Title, Body and Footer. The Title and Body are mandatory if the description is large enough. The footer is optional.
+
+#### The command :
+
+git commit -m "`type_of_commit(optional scope)`: `description_title/description`" -m "`description body(optional)`" -m "`description footer(optional)`"
+
+#### The Example:
+
+git commit -m "`docs(guides)`: `change in how to write professional commit`" -m "`fix spelling mistake, add example`" -m "`fixes issue #120`"
```

Comment:
This is what we wanted as an example.
Looks Good 🎉

Reply:
> This is what we wanted as an example. Looks Good 🎉

I am really glad that you liked it.